build.sh

```diff
@@ -14,6 +14,7 @@ fi
 buf generate
 
 go build -ldflags "\
+  -extldflags=-static
```

Comment:
Don't need static compiling.

Reply:
Added a `EXT_LD_FLAGS` env var in build.sh and set it `-static` only in distroless Dockerfile.